<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-2008 Trolltech ASA. All rights reserved.
**
** This file is part of the Qt Designer of the Qt Toolkit.
**
** This file may be used under the terms of the GNU General Public
** License versions 2.0 or 3.0 as published by the Free Software
** Foundation and appearing in the files LICENSE.GPL2 and LICENSE.GPL3
** included in the packaging of this file.  Alternatively you may (at
** your option) use any later version of the GNU General Public
** License if such license has been publicly approved by Trolltech ASA
** (or its successors, if any) and the KDE Free Qt Foundation. In
** addition, as a special exception, Trolltech gives you certain
** additional rights. These rights are described in the Trolltech GPL
** Exception version 1.2, which can be found at
** http://www.trolltech.com/products/qt/gplexception/ and in the file
** GPL_EXCEPTION.txt in this package.
**
** Please review the following information to ensure GNU General
** Public Licensing requirements will be met:
** http://trolltech.com/products/qt/licenses/licensing/opensource/. If
** you are unsure which license is appropriate for your use, please
** review the following information:
** http://trolltech.com/products/qt/licenses/licensing/licensingoverview
** or contact the sales department at sales@trolltech.com.
**
** In addition, as a special exception, Trolltech, as the sole
** copyright holder for Qt Designer, grants users of the Qt/Eclipse
** Integration plug-in the right for the Qt/Eclipse Integration to
** link to functionality provided by Qt Designer and its related
** libraries.
**
** This file is provided "AS IS" with NO WARRANTY OF ANY KIND,
** INCLUDING THE WARRANTIES OF DESIGN, MERCHANTABILITY AND FITNESS FOR
** A PARTICULAR PURPOSE. Trolltech reserves all rights not expressly
** granted herein.
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>qdesigner_internal::TreeWidgetEditor</class>
 <widget class="QDialog" name="qdesigner_internal::TreeWidgetEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Tree Widget</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="itemsBox" >
     <property name="title" >
      <string>Tree Items</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="pixmapLabel" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Pixmap</string>
          </property>
          <property name="buddy" >
           <cstring>previewPixmapItemButton</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="previewPixmapItemButton" >
          <property name="toolTip" >
           <string>Set Item Pixmap</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deletePixmapItemButton" >
          <property name="toolTip" >
           <string>Reset Item Pixmap</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Text</string>
          </property>
          <property name="buddy" >
           <cstring>itemTextLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="itemTextLineEdit" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>13</hsizetype>
            <vsizetype>4</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Set Item Text</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="newItemButton" >
          <property name="toolTip" >
           <string>New Item</string>
          </property>
          <property name="text" >
           <string>&amp;New Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="newSubItemButton" >
          <property name="toolTip" >
           <string>New Subitem</string>
          </property>
          <property name="text" >
           <string>New &amp;Subitem</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteItemButton" >
          <property name="toolTip" >
           <string>Delete Item</string>
          </property>
          <property name="text" >
           <string>&amp;Delete Item</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="moveItemLeftButton" >
          <property name="toolTip" >
           <string>Move Item Left (before Parent Item)</string>
          </property>
          <property name="text" >
           <string>L</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="moveItemRightButton" >
          <property name="toolTip" >
           <string>Move Item Right (as a First Subitem of the Next Sibling Item)</string>
          </property>
          <property name="text" >
           <string>R</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" >
       <widget class="QToolButton" name="moveItemUpButton" >
        <property name="toolTip" >
         <string>Move Item Up</string>
        </property>
        <property name="text" >
         <string>U</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QToolButton" name="moveItemDownButton" >
        <property name="toolTip" >
         <string>Move Item Down</string>
        </property>
        <property name="text" >
         <string>D</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QTreeWidget" name="treeWidget" >
        <property name="focusPolicy" >
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Tree Items</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="columnsBox" >
     <property name="title" >
      <string>Columns</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="pixmapColumnLabel" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Pixmap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="previewPixmapColumnButton" >
          <property name="toolTip" >
           <string>Set Column Pixmap</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deletePixmapColumnButton" >
          <property name="toolTip" >
           <string>Reset Column Pixmap</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Ignored</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="newColumnButton" >
          <property name="toolTip" >
           <string>New Column</string>
          </property>
          <property name="text" >
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="renameColumnButton" >
          <property name="toolTip" >
           <string>Rename Column</string>
          </property>
          <property name="text" >
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteColumnButton" >
          <property name="toolTip" >
           <string>Delete Column</string>
          </property>
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Ignored</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>29</width>
            <height>23</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1" >
       <widget class="QToolButton" name="moveColumnUpButton" >
        <property name="toolTip" >
         <string>Move Column Up</string>
        </property>
        <property name="text" >
         <string>U</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QToolButton" name="moveColumnDownButton" >
        <property name="toolTip" >
         <string>Move Column Down</string>
        </property>
        <property name="text" >
         <string>D</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QListWidget" name="listWidget" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>13</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Tree Columns</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>newItemButton</tabstop>
  <tabstop>newSubItemButton</tabstop>
  <tabstop>deleteItemButton</tabstop>
  <tabstop>moveItemLeftButton</tabstop>
  <tabstop>moveItemRightButton</tabstop>
  <tabstop>moveItemUpButton</tabstop>
  <tabstop>moveItemDownButton</tabstop>
  <tabstop>previewPixmapItemButton</tabstop>
  <tabstop>deletePixmapItemButton</tabstop>
  <tabstop>itemTextLineEdit</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>newColumnButton</tabstop>
  <tabstop>renameColumnButton</tabstop>
  <tabstop>deleteColumnButton</tabstop>
  <tabstop>moveColumnUpButton</tabstop>
  <tabstop>previewPixmapColumnButton</tabstop>
  <tabstop>deletePixmapColumnButton</tabstop>
  <tabstop>moveColumnDownButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qdesigner_internal::TreeWidgetEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>450</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qdesigner_internal::TreeWidgetEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>539</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel" >
     <x>556</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
